src/components/ebay-badge/README.md

```diff
@@ -1,4 +1,4 @@
-# ebay-badge
+# ebay-badge v1.2.0
```

Comment:
As discussed offline, let's prefix the version number with "DS". It would be great if we could reduce it's visual prominence somehow - perhaps by reducing the font-size and/or aligning it over to the right, if possible.

Reply:
Yeah I was thinking we could do something like `<div style="text-align: right; font-size: small;"> DS v1.2.0 </div>`

Reply:
Does Storybook/Markdown support classes instead? e.g. `<div class="ds-version">DS v1.2.0 </div>`

Reply:
I don't think they do unless you add some third party libraries. I think in this case the inline styles would be alright because it's just in the markdown file.